plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/build.properties

```diff
@@ -4,3 +4,4 @@ bin.includes = META-INF/,\
                .
 javacSource=1.7
 javacTarget=1.7
+additional.bundles = org.eclipse.emf.ecore
```

Comment:
I wonder if we can remove this package from MANIFEST.MF then.

Some question: if I remove the package from MANIFEST.MF, I get an error marker "The type org.eclipse.emf.ecore.EObject cannot be resolved. It is indirectly referenced from required .class files". I learned that I can remove the error by adding the package to MANIFEST.MF, but I've always wondered why the quick fix in this case doesn't offer to add it to MANIFEST.MF. Is this the case where I need to put it through `additional.bundles`?